lib/declaration.ts

```diff
@@ -963,48 +958,106 @@ export default class DeclarationTranspiler extends base.TranspilerBase {
     this.emitNoSpace(';');
   }
 
-  private emitMethodsAsExtensions(
-      className: ts.Identifier, visitName: () => void, visitNameOfExtensions: () => void,
+  private emitGetterOrSetter({mode, declaration, emitJsAnnotation, isExternal, emitBody}:
```

Comment:
provide types for these arguments.

Reply:
The first part is just specifying the names for object destructuring the types are defined by `:emitGetterOrSetterOptions`